base/tks/src/org/dogtagpki/server/tks/servlet/TokenServlet.java

```diff
@@ -1872,7 +1890,8 @@ private void processDiversifyKey(HttpServletRequest req,
                             nistSP800_108KdfUseCuidAsKdd, // AC: KDF SPEC CHANGE - pass in configuration file value
                             xCUID, // AC: KDF SPEC CHANGE - removed duplicative 'CUID' variable and replaced with 'xCUID'
                             xKDD, // AC: KDF SPEC CHANGE - pass in KDD so symkey can make decision about which value (KDD,CUID) to use
-                            kekKeyArray,encKeyArray,macKeyArray, useSoftToken_s, keySet, (byte) protocol,gp3Params);
+                            kekKeyArray,encKeyArray,macKeyArray, useSoftToken_s, keySet, (byte) protocol,gp3Params, 
+                            oldGp3Params);  // ** G&D 256 Key Rollover Support **  add oldGp2Params parameter to the method call
```

Comment:
typo? I think the patch meant to say "... add oldGp3arams parameter ..." instead of oldGp2params

Reply:
Yes, that looks like a typo, we don't support scp02 and more and don't have any Gp2 params class.

Reply:
Pushed PR again to fix typo.